api/index.ts

```diff
@@ -76,13 +80,16 @@ async function apiBuilder (cfg: APIConfig): Promise<FastifyTypebox> {
   await registerAuth(cfg, server, store)
   await server.register(multipart)
 
+  /*
   // pre-sync all sites
   const allSites = await store.sites.keys()
-  await Promise.all(allSites.map(async (siteId) => {
+  await pMap(allSites, async (siteId) => {
     server.log.info(`Presyncing site: ${siteId}`)
     const fp = store.fs.getPath(siteId)
     await store.sites.sync(siteId, fp, { logger: server.log })
-  }))
+    server.log.info(`Finished presync: ${siteId}`)
+  }, { concurrency: 1 })
+  */
```

Comment:
Is this ready for review? I wonder if we can queue an async job *after* boot to do this step (i.e. this shouldn't block boot)